webrtc.html

```diff
@@ -866,7 +866,7 @@
 
                   <li>
                     <p>If <var>connection</var>'s <a>signaling state</a>
-                    changed above, fire a simple event named <code><a href=
+                    changed above, queue a task to fire a simple event named <code><a href=
```

Comment:
@alvestrand are you arguing we should leave things as they are (i.e. we queue a task that potentially changes the state, and if it does it fires the event rather than queuing another task to fire the event)?